src/node.js

```diff
@@ -71,6 +71,7 @@ const Node = AsyncInit.compose({
     })
     this.version = client.spec.info.version
     if (
+      !this.version.startsWith('0.0.0-hc') &&
```

Comment:
Isn't it better the clients to bypass the check with `ignoreVersion` instead of this ugly looking hardcoded version/tag bypass ?

Reply:
I would replace it with a specific version check instead of `startsWith`. I think it is fine to keep it as far as sdk being compatible with this version.